app/controllers/api/v1/accounts_controller.rb

```diff
@@ -4,7 +4,7 @@ class Api::V1::AccountsController < ApiController
   before_action -> { doorkeeper_authorize! :read }, except: [:follow, :unfollow, :block, :unblock, :mute, :unmute]
```

Comment:
With how these are laid out, `AccountsController#update_credentials` would require only "read" scope from apps. It should be "write", or perhaps even a new type of scope altogether, since no other API methods have allows you to modify your profile before